src/main/java/org/opensearch/security/auth/BackendRegistry.java

```diff
@@ -138,13 +137,15 @@ public BackendRegistry(
         this.opensearchSettings = settings;
         this.xffResolver = xffResolver;
         this.auditLog = auditLog;
-        this.threadPool = threadPool;
         this.userInjector = new UserInjector(settings, threadPool, auditLog, xffResolver);
+        this.restAuthDomains = Collections.emptySortedSet();
+        this.ipAuthFailureListeners = Collections.emptyList();
```

Comment:
Where will this variable be utilised?

Reply:
I am going to revert this change. I was working through the SystemIntegratorTests and ended up resolving it another way.